Premiere/15.0/index.d.ts

```diff
@@ -2010,26 +2010,40 @@ declare class ComponentParamCollection {
 
 declare class ComponentParam {
   readonly displayName: string
-  addKey(): boolean
+  addKey(time: Time, updateUI?: boolean): boolean
   areKeyframesSupported(): boolean
-  findNearestKey(): object
-  findNextKey(): object
-  findPreviousKey(): object
+  /**
+   * threshold in ticks
+   */
+  findNearestKey(timeToCheck: Time, threshold: number): object
+  findNextKey(timeToCheck: Time): object
```

Comment:
`time to check` doesn't add much context / explanation. Better to leave it as `time`
```suggestion
  findNextKey(time: Time): object
```